modules/wmi/charts.go

```diff
@@ -69,7 +69,14 @@ const (
 	prioProcessesPoolBytes
 	prioProcessesThreads
 
-	prioServiceState
+	prioServiceStateContinuePending
+	prioServiceStatePausePending
```

Comment:
I don't think this will do. My suggestion is:
- a chart per service
- the chart dimensions - states.

Reply:
this will be unusable, because we will have hundred/thousands  of charts.. :/

Reply:
what you are doing is unusable too

Reply:
We need to do something like we are doing right now with Linux services.

Reply:
> we will have hundred/thousands of charts

As I said - ND Cloud UI (overview)

Reply:
> We need to do something like we are doing right now with Linux services.

Not possible. Prove me wrong.

Reply:
Windows has hundred of services, times 12 different status we will have thousands, this is the motive I think this is the safest way to go with Microsoft.

Reply:
No. The number of charts == the number of services * the number of metrics. It is not a safe/unsafe question - we don't really have a lot of options here.